mapquester_frontend/src/app/map/_components/MapComponent.tsx

```diff
@@ -2,20 +2,22 @@ import { useState, useCallback, useRef } from 'react';
 import Map, { Marker, ViewState, MapRef, MapMouseEvent } from 'react-map-gl';
 import 'mapbox-gl/dist/mapbox-gl.css';
 import POIForm from './POIForm';
+import UpdatePOIForm from './UpdatePOIForm';
 
 interface Point {
   name: string;
   longitude: number;
   latitude: number;
   description: string;
+  tag: string;
```

Comment:
let's make tag an enum that can be selected from a list. for now add:
- food
- event
- school
- photo
- music

Reply:
They can be options that are available by default but let's not restrict the tag to only take on these values.

User must be able to create custom tags.

Reply:
@ysjprojects for the sake of limiting scope, let's have default for now. We can look to extend it later by adding categories or some other value.